### PR TITLE
Update to emc_recipies

### PR DESCRIPTION
**Pull request by DarkOakMetarex**
Adds: Popped Chorus Fruit is an end-game item used to craft End Rods and Purpur Blocks that was added in snapshot 15w31a of the 1.9 update.

Notes: This could serve as an alternate recipe for end rods and or be an additional way to craft them.

### CODE REVIEW

File: src/main/java/com/karlofduty/EMCRecipies/EMCRecipies.java [hunk 1]
```diff
@@ -197,5 +197,15 @@ public void onEnable()
 
         this.getServer().addRecipe(new FurnaceRecipe(new ItemStack(Material.SMOOTH_SANDSTONE, 1), Material.SANDSTONE));
         this.getServer().addRecipe(new FurnaceRecipe(new ItemStack(Material.SMOOTH_RED_SANDSTONE, 1), Material.RED_SANDSTONE));
+        
+        /*
+            POPPED CHORUS FRUIT
+         */
+
+        final ShapedRecipe poppedchorusfruit = new ShapedRecipe(new NamespacedKey(this, "emc_poppedchorusfruit"), new ItemStack(Material.POPPED_CHORUS_FRUIT, 4));
```
Comment (KarlOfDuty) [inline src/main/java/com/karlofduty/EMCRecipies/EMCRecipies.java]: Should this really be a shaped recipe?

Comment (DarkOakMetarex) [reply src/main/java/com/karlofduty/EMCRecipies/EMCRecipies.java]: No it should not have been, that was my mistake, an oversight.

**Comment by KarlOfDuty** [comment]
Haven't you tested any of this? The plugin doesn't even compile?

**Comment by DarkOakMetarex** [comment]
I'm not qualified to run testing, I have only 3 years of coding experience and it's not in Java.

**Comment by jkmartindale** [comment]
> I'm not qualified to run testing, I have only 3 years of coding experience and it's not in Java.

You don't have to be qualified, it just takes a little practice. Never too late to start :)

**Comment by KarlOfDuty** [comment]
I'm not saying you'd have to do some advanced testing, just try using the crafting recipes you added to make sure they work.

**Pull request merged**